### DISCUSSION TITLE
Upload/Download files within code-server in Explorer

### DISCUSSION THREAD

**Discussion by oxFilla** [coder/code-server#2136] [Q&A] [answered]
How can I Upload/Download files within code-server in Explorer? Seems only possible through terminal or cut and past text in new files.

Is there an easier way directly in Explorer?

**Comment by code-asher** [comment, marked as answer]
You can upload files using drag and drop. There isn't a way to download files yet though. We have an open issue for that here: https://github.com/cdr/code-server/issues/1386

**Reply by oxFilla** [reply]
Ah, good to know. Thank you very much.
